superx/information_extractor.py

```diff
@@ -0,0 +1,131 @@
+
+'''
+    This is in charge of extracting the XML document
+    XML files are found @ https://www.consumers.org.il/item/transparency_price
+'''
+import mysql.connector
+import requests
+from bs4 import BeautifulSoup
+from datetime import datetime
+import gzip
+import xml.etree.ElementTree as ET
+
+# url list with category PriceFull
+url_list = ['http://publishprice.mega.co.il/%s' % datetime.today().strftime('%Y%m%d'),  # mega link with correct date
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=1',
```

Comment:
All the shufersal links are the same aside from the page. Instead of listing each page individually here, other ways to do it would be to have an additional variable that says how many pages, or see what happens when you get to a non-existent page and look for that error.
You can also have a dictionary/variable for each chain defining if it is a single page or multiple pages that need to be scrolled through

Reply:
there are 23 pages, if for instance i ask for page 55 then i get page 23, so there is no harm done asking for more pages than given. Ive checked on multiple days and all have had 23 pages. I could make an external method that all it does is add 3 shufersal url's to the list, but i feel that this is redundant. The urls are constants and so why not build a list that is created with its variables?

Reply:
with regards to the rest, they use only 1 page. (the rest meaning the other links that i have implemented into the code

Reply:
I'm not going to hold up the PR for this, but it's not good looking code. In this case, most probably the number of pages won't change, but they might at some point in time and you wouldn't know.

Reply:
ok i understand and ill think of a way to make it more general